website/docs/tutorials/07-testing/02-cheatcodes/example.md

```diff
@@ -18,6 +18,9 @@ This cheatcode is only available in [setup cases](../README.md#setup-case).
 :::warning
 `example` is not limited by [`max_examples`](./max-examples.md) and is not connected to it in any way.
 :::
+:::warning
+You have to provide the testing data explicitly either by using `example` or [`given`](./given.md).
+:::
```

Comment:
This is a weird information in current context.
```suggestion

```

Reply:
[Address direct comments](https://github.com/software-mansion/protostar/pull/995/commits/d53af654b1789f34245a4e363740e3035ff1cfb5)